cmake/configs/aarch64/darwin/config.cmake

```diff
@@ -34,3 +34,8 @@ if(has_arm_neon)
     add_cmake_flag(CMAKE_C_FLAGS -DHAVE_ARM_NEON_INTRIN_H)
 endif()
 unset(has_arm_neon CACHE)
+
+# Header file here is required for portable futex implementation.
+if(NOT CMAKE_CROSSCOMPILING)
```

Comment:
Why the `CMAKE_CROSSCOMPILING` check?

Reply:
@marcbutler This is still an open question from the last round.

Reply:
Sorry thought I'd covered everything.
I'd just as soon leave it as-is; because it reflects the structure of other similar entries. It wouldn't be suitable to include this file if you were say cross compiling for linux on a mac.
I'm happy to create a ticket to remove `CMAKE_CROSSCOMPILING` support which I suspect does not work. Does that work for you?

Reply:
Yep, I'd be happy with that - thanks.

Reply:
Created: [WT-13192](https://jira.mongodb.org/browse/WT-13192)